docs/usage/help.rst.inc

```diff
@@ -120,7 +120,7 @@ Examples::
     # Comment line
     /home/*/junk
     *.tmp
-    fm:aa:something/*
+    fm:something/*
```

Comment:
that seems like a good catch.

strange how that "aa:" crept in here.

Reply:
I just searched for `aa:` and found this in the docs:
```
If followed by a colon (':') the first two characters of a pattern are used as a
style selector. Explicit style selection is necessary when a
non-default style is desired or when the desired pattern starts with
two alphanumeric characters followed by a colon (i.e. `aa:something/*`).
```

So yet another weird example showing how to exclude something that starts with aa: (or 2 letters plus a colon, in general).

Reply:
Again, sorry